### ISSUE TITLE
Ensures ARIA attributes are allowed for an element's role (#_weave_e_22554).

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1025]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216285

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" Menu item and select it.
Navigate through all the controls under "Data Explorer" blade.
Run accessibility Insight for web tool and select 'fast pass' option.
Verify the issue.

Actual Result:
ARIA attribute cannot be used, add a role attribute or use a different element: aria-label

Expected Result:
Ensures ARIA attributes are allowed for an element's role (#_weave_e_22554).

Issue: 
Ensures ARIA attributes are allowed for an element's role (aria-allowed-attr - https://accessibilityinsights.io/info-examples/web/aria-allowed-attr) 
 

Target application: 
shstoretest - Microsoft Azure - https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/227b734f-e14f-4de6-b7fc-3190c21e69f6/resourceGroups/shstorerc/providers/Microsoft.DocumentDb/databaseAccounts/shstoretest/dataExplorer

Element path: 
#_weave_e_22554


Snippet: 
<div class="fxs-blade-content-wrapper" id="_weave_e_22554" aria-label="Data Explorer">


How to fix:
Fix all of the following:
ARIA attribute cannot be used, add a role attribute or use a different element: aria-label


Environment:
Chrome version 91.0.4472.77

![image](https://user-images.githubusercontent.com/8519407/132721055-28aba63c-b18b-47fa-b831-82e3f93495cd.png)
